.github/workflows/TestingCI.yml

```diff
@@ -9,20 +9,21 @@ env:
 
 jobs:
   linux-ubuntu:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v4
-    - name: Fetch
-      run: cargo fetch
-    - name: Build
-      run: cargo build --release --verbose
-    - name: Run tests
-      run: cargo test --release --verbose
+      - uses: actions/checkout@v4
+      - name: Fetch
+        run: cargo fetch
+      - name: Build
+        run: cargo build --release --verbose
+      - name: Run tests
+        run: cargo test --release --verbose
+      - name: Run fmt check
+        run: cargo fmt --all -- --check
+#      - name: Run clippy check
```

Comment:
1) A PR should "do what it says, and no more"

2) Never add commented-out code or instructions.  Any code not run is code never tested, and will become fragile and break.

3) Delete the commented out part of this PR, and it is ok.